pkg/openSearch/openSearchClient/config/config.go

```diff
@@ -24,6 +24,15 @@ func ReadOpensearchClientConfig() (OpensearchClientConfig, error) {
 	if err != nil {
 		return *config, err
 	}
-	log.Debug().Msgf("OpenSearch Config: %+v", config)
+
+	log.Debug().
+		Str("opensearch_client_config_host", config.Host).
+		Int("opensearch_client_config_port", config.Port).
+		Bool("opensearch_client_config_https", config.Https).
+		Str("opensearch_client_config_auth_username", config.AuthUsername).
+		Str("opensearch_client_config_auth_password", config.AuthPassword).
+		Str("opensearch_client_config_auth_method", config.AuthMethod).
+		Msgf("OpenSearch Config: %+v", config)
```

Comment:
```suggestion
		Msgf("OpenSearch Config")
```
already supplied separately. If the attributes should not be flattened out you could use `.Any("config",config)` instead of the separate properties

Reply:
resolved, should remain in `Msgf`, see [comment](https://github.com/greenbone/opensight-golang-libraries/pull/95#discussion_r1696699393)